phoenix-tracing-webapp/src/main/config/checkstyle/checker.xml

```diff
@@ -27,8 +27,8 @@ limitations under the License.
 -->
 
 <!DOCTYPE module PUBLIC
-        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
```

Comment:
Updated based on comment in file http://www.puppycrawl.com/dtds/configuration_1_3.dtd, let see if this helps

Reply:
its possible this may fix it as we were using http before and now it changes to https as mentioned by someone in https://stackoverflow.com/questions/44333939/the-markup-declarations-contained-or-pointed-to-by-the-document-type-declaration. also possible this check was never passing? lets see what build says

Reply:
Looks good according to the tests.

Reply:
> Looks good according to the tests.
> 

Yes this got fixed now. Thankfully :) I am assuming the XML test was never triggered for a long time and was broken for a while.